test/package/browser/template/src/index-modular.ts

```
@@ -2,6 +2,12 @@ import { BaseRealtime, WebSocketTransport, FetchRequest, generateRandomKey } fro
 import { InboundMessage, RealtimeChannel } from 'ably';
 import { createSandboxAblyAPIKey } from './sandbox';
 
+// Fix for "type 'typeof globalThis' has no index signature" error:
+// https://stackoverflow.com/questions/68481686/type-typeof-globalthis-has-no-index-signature
+declare module globalThis {
```

Biome (1.9.4)

[error] Use the namespace keyword instead of the outdated module keyword.

The module keyword is deprecated to avoid any confusion with the ECMAScript modules which are often called modules.
Safe fix: Use namespace instead.


(lint/suspicious/useNamespaceKeyword)